orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/config/PreconfiguredWebhookProperties.groovy

```diff
@@ -67,6 +68,11 @@ class PreconfiguredWebhookProperties {
     }
   }
 
+  static class WebhookParameter {
+    String label
+    String defaultValue
```

Comment:
The other ticket also mentions a `type` ... that's probably also relevant here.

I could imagine drop-down's, etc.

Reply:
I thought about implementing that in this PR, but wanted to get the basics in first. I suppose I could add that and default to string type. other types could be added later.